kolibri/plugins/learn/assets/src/vue/index.vue

```diff
@@ -40,8 +40,6 @@
   const PageModes = constants.PageModes;
   const getters = require('../state/getters');
   const store = require('../state/store');
-  require('vue-scroll');
-
 
```

Comment:
@whitzhu: Since you are already editing this file, change the line 8 here 

from
`<label for="chan-select" class="visuallyhidden">Filter User Type</label>`
to 
`<label for="chan-select" class="visuallyhidden">Select Channel</label>`

Somebody was probably too eager at copy/pasting... :stuck_out_tongue: